prime-router/docs/openapi.yml

```diff
@@ -9,6 +9,380 @@ info:
   version: 0.2.0-oas3
 paths:
 
+  # The sender end-point handles receive actions.
+  #
+  # END-POINTS
+  # /submissions
+  # /submissions/multorg
+  # /submissions/{reportid}
+  # /submissions/{organizationName}
+  #
+  # FILTERS (OPTIONAL)
+  # ?offset=INT
+  # ?limit=INT
+  # ?dateStart=STRING (YYYY-MM-DD) || INT (NN [days back] | default: 30)
+  # ?dateEnd=STRING (YYYY-MM-DD | default: Today)
+  # ?year=INT (YYYY | default: ALL)
+  # ?month=INT (MM | default: ALL | requires ?year)
+  # ?organizations=INT (default: ALL)
+  # ?recipients=ARRAY
+  # ?sender=ARRAY
+  # ?warnings=BOOLEAN
+  # ?errors=BOOLEAN
+  # ?receipts=BOOLEAN
+  #
```

Comment:
Another useful feature for client the client UI is to have something like a `?getlength=true` that just returns the `count()`
(e.g. It's used when building the pagination UI to show the total number of pages.)